### PR TITLE
SR1.1.1 Add support for showing daily standard deviation on inflammation data

### PR DESCRIPTION
**Pull request by davidmeijer**
Extend the graphs to show the daily standard deviation as required by SR1.1.1

### CODE REVIEW

File: inflammation-analysis.py [hunk 1]
```diff
@@ -21,7 +21,7 @@ def main(args):
     for filename in InFiles:
         inflammation_data = models.load_csv(filename)
 
-        view_data = {'average': models.daily_mean(inflammation_data), 'max': models.daily_max(inflammation_data), 'min': models.daily_min(inflammation_data)}
+        view_data = {'average': models.daily_mean(inflammation_data), 'max': models.daily_max(inflammation_data), 'min': models.daily_min(inflammation_data), **(models.s_dev(inflammation_data))}
```
Comment (Anna-MarieSeelen) [inline inflammation-analysis.py:24]: In this line I think the ** is not in line with the formatting of the other variables. I suggest the following change:
```suggestion
        view_data = {'average': models.daily_mean(inflammation_data), 'max': models.daily_max(inflammation_data), 'min': models.daily_min(inflammation_data), 'sd':(models.s_dev(inflammation_data))}
```

File: inflammation/models.py [hunk 2]
```diff
@@ -32,3 +32,13 @@ def daily_min(data):
     """Calculate the daily min of a 2d inflammation data array."""
     return np.min(data, axis=0)
 
+
+def s_dev(data):
+    """Computes and returns standard deviation for data."""
+    mmm = np.mean(data, axis=0)
```
Comment (Anna-MarieSeelen) [inline inflammation/models.py:38]: I think mmm is an unclear variable name, but maybe its short for something I don't know?

File: inflammation/models.py [hunk 3]
```diff
@@ -32,3 +32,13 @@ def daily_min(data):
...
+    devs = []
+    for entry in data:
+        devs.append((entry - mmm) * (entry - mmm))
+
+    s_dev2 = sum(devs) / len(data)
+    return {'standard deviation': s_dev2}
```
Comment (Anna-MarieSeelen) [inline inflammation/models.py:44]: Why do you return a dictionary? Why not just the value?

File: inflammation/models.py (hunk 2 above)
Comment (Anna-MarieSeelen) [reply inflammation/models.py:38]: Maybe mean is better

**Review by Anna-MarieSeelen — CHANGES_REQUESTED** [review]
Good luck!

File: inflammation-analysis.py (hunk 1 above)
Comment (davidmeijer) [reply inflammation-analysis.py:24]: That's a great suggestion! Thanks!

File: inflammation/models.py (hunk 3 above)
Comment (davidmeijer) [reply inflammation/models.py:44]: Good point!

**Pull request merged**